samples/azure/includes/asb-connectionstrings.include.md

```diff
@@ -0,0 +1 @@
+2 environment variables named `AzureServiceBus.ConnectionString1` and `AzureServiceBus.ConnectionString2` with a different connection string to an Azure Service Bus namespace each.
```

Comment:
Does this have to be an include file if this is the only sample that requires it?